docs/tutorial/fdcurves.rst

```diff
@@ -72,6 +107,12 @@ and distance from such an ensemble using::
 
     f = fd_ensemble.f
     d = fd_ensemble.d
+    plt.scatter(d, f, s=1)
+    plt.ylabel("Force (pN)")
+    plt.xlabel("Distance $\mu$m")
+    plt.title("Two aligned fd curves")
+
+.. image:: figures/fdcurves/fdcurves_aligned.png
 
 Baseline Correction
```

Comment:
You didn't change this, but I noticed that the indent below is a bit strange. Normal convention is two spaces after the code, and one after the hash.

```python
    file = lk.File("example.h5")
    fd = file.fdcurves["baseline"]  # low frequency, uncorrected force magnitude
    fd_bl = fd.with_baseline_corrected_x()  # low frequency, baseline corrected force x-component
```

Also, it might be nice to move the note _after_ the baseline correction is explained? Otherwise you start reading it before even knowing about baseline correction.

Reply:
Good point, included the suggestions